Game.py

```
@@ -1,79 +1,192 @@
+
+# ====== Menu chọn level ======
+def show_level_menu(level_count):
+    print("Chọn level để chơi:")
+    for i in range(level_count):
+        print(f"{i + 1}. Level {i + 1}")
+    while True:
+        try:
+            choice = int(input("Nhập số level: "))
+            if 1 <= choice <= level_count:
+                return choice - 1
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)